policy/services.py

```diff
@@ -37,6 +37,27 @@ def reset_policy_before_update(policy):
     policy.officer_id = None
 
 
+class PolicyRenewalService:
+    def __init__(self, user):
+        self.user = user
+
+    def delete(self, policy_renewal):
+        try:
+            policy_renewal = PolicyRenewal.objects.get(uuid=policy_renewal.uuid)
```

Comment:
Maybe it's a dumb question, but why do you search for the `policy_renewal` instance if you already receive it as a parameter? Or maybe these are 2 different things with the same name?

Reply:
Yes!